.github/workflows/npmpublish.yml

```diff
@@ -1,38 +1,39 @@
-name: "NPM Publish and GitHub tag"
+name: 'NPM Publish and GitHub tag'
 on:
   push:
     branches:
       - main
 
 jobs:
-  build:
+  publish:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3.1.0
       - uses: ./.github/actions/pnpm-setup-action
       - run: pnpm -r publish --access public
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
-      # Set PACKAGE_VERSION, pull from package.json
-      - name: package-version
-        run: node -p -e '`PACKAGE_VERSION=${require("./package.json").version}`' >> $GITHUB_ENV
-
-      # Now check if the synapse-react-client version changed, and if so, create a tag.
-      # We will need to revise/generalize this workflow if we publish multiple packages to npm.
-      - id: synapse-react-client-version-check
+  tag:
+    strategy:
+      matrix:
+        publishablePackage: [synapse-react-client, markdown-it-synapse]
```

Comment:
for now, enumerate all published packages so we know where to look for version bumps